### ISSUE TITLE
Ubuntu 21.10 user creation in image not making connection to local machine

### ISSUE THREAD

**Issue by IncrediblePony** [bitnami/bitnami-docker-laravel#161]
### Name and Version

bitnami/laravel

### What steps will reproduce the bug?

In Ubuntu 21.10 when having a fresh install of docker and docker compose I downloaded the docker-compose.yml file as described on docker hub. The docker container runs when the command `docker compose up` has been issued.

When running `docker exec <myappname> php artisan make:seed MyTestSeeder` the file is created at the expected location and everything so far is fine.

When opening the application in any IDE or text editor I cannot write to the created file (`MyTestSeeder`) due to the fact that that the file has been created by the root user of the container and passed on to the root user of my local machine and by design I cannot write to that file in any way. I could do a `chown -R xxx` on my working directory but then I would have to do this every time I generated a file (model/view/controller/seeder/migration) with php artisan.

I have also tried to specify a user in the `docker-compose.yml` file based on my local user on my machine. But this results in an error because I do not have the same user specified in my containers OS. And now I'm stuck.

### What is the expected behavior?

I would expect the if I in my `docker-compose.yml` file specified a user as such:

```yaml
version: '2'

services:
  mariadb:
    image: docker.io/bitnami/mariadb:10.6
    environment:
      # ALLOW_EMPTY_PASSWORD is recommended only for development.
      - ALLOW_EMPTY_PASSWORD=yes
      - MARIADB_USER=bn_myapp
      - MARIADB_DATABASE=bitnami_myapp
  myapp:
    user: '$USER:$USER'
    image: docker.io/bitnami/laravel:9
    ports:
      - '8000:8000'
    environment:
      - DB_HOST=mariadb
      - DB_PORT=3306
      - DB_USERNAME=bn_myapp
      - DB_DATABASE=bitnami_myapp
    volumes:
      - './my-project:/app'
    depends_on:
      - mariadb
```

That the container on the initial run/startup would check if that user existed and if not, the user would be created with the correct permissions within the container.

### What do you see instead?

Instead when I specify a user in the config I get the output from the container that `Error response from daemon: unable to find user mads: no matching entries in passwd file` or if the usergroup in the container is not matching my local machines group: `Error response from daemon: unable to find group mads: no matching entries in group file`

### Additional information
https://github.com/aschmelyun/docker-compose-laravel/blob/main/docker-compose.yml
https://github.com/aschmelyun/docker-compose-laravel/blob/main/dockerfiles/nginx.dockerfile

An example on how to do this.

EDIT: added additional info

**Comment by carrodher** [comment]
Please, [note the Bitnami team is not actively monitoring GitHub Issues for this project](https://github.com/bitnami/bitnami-docker-laravel#community-supported-solution). This issue will stay open for 20 days to allow the community to contribute, after 15 days without activity the issue will be marked as stale being closed after 5 days.

If you would like to contribute to this container image by fixing a bug or adding a new feature feel free to create a PR, the Bitnami team will be happy to review it and provide feedback.

**Comment by IncrediblePony** [comment]
https://github.com/aschmelyun/docker-compose-laravel/blob/main/docker-compose.yml
https://github.com/aschmelyun/docker-compose-laravel/blob/main/dockerfiles/nginx.dockerfile

An example on how to do this.

**Comment by IncrediblePony** [comment]
A further update. I have now managed to "circumvent" the issue, but I still do not think that this should be the default behaviour for the container. It still should be set up so that if a $USER:$USER is specified in the `docker-compose.yml` file then run a script inside the container adding the user by the hosts username and group. If it's two id's that specified then create a user with those ids if they don't exist.

On my own machine I modified my user and usergroup id's to be `1000:1000` since I, by accident, stumbled over your `bitnami-user.sh` file in this repo (https://github.com/bitnami/bitnami-docker-laravel/blob/master/9/debian-10/prebuildfs/build/bitnami-user.sh) since these id's match the id's set by the containers OS be default. 

Now I can specify the `user: '1000:1000` setting in the `docker-compose.yml` and write to the files created by the `php artisan` commands on my host machine.
